Ginger/GingerCoreCommon/Repository/ApplicationModelLib/POMModelLib/ApplicationPOMModel.cs

```diff
@@ -120,7 +120,7 @@ public ObservableList<ElementInfo> UnMappedUIElements
         /// <summary>
         /// Been used to identify if MappedUIElements were lazy loaded already or not
         /// </summary>
-        public bool MappedUIElementsLazyLoad { get { return mMappedElements.LazyLoad; } }
+        public bool MappedUIElementsLazyLoad { get { return (mMappedElements != null) ? mMappedElements.LazyLoad : false; } }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5022389)